index.html

```diff
@@ -0,0 +1,401 @@
+                  <p class="nft-card__price">$199</p>
+                  <h3 class="nft-card__title">NAME PRODUCT</h3>
+                </article>
+              </a>
+            </li>
+            <li class="nft-card">
+              <a href="#" class="nft-card__link">
+                <article class="nft-card__content">
+                  <img class="nft-card__image" src="#" alt="NFT Item 2">
+                  <span class="nft-card__badge nft-card__badge--new">NEW</span>
+                  <p class="nft-card__price">$199</p>
+                  <h3 class="nft-card__title">NAME PRODUCT</h3>
+                </article>
+              </a>
+            </li>
+            <li class="nft-card">
+              <a href="#" class="nft-card__link">
+                <article class="nft-card__content">
+                  <img class="nft-card__image" src="#" alt="NFT Item 3">
+                  <span class="nft-card__badge nft-card__badge--sold-out">SOLD OUT</span>
+                  <p class="nft-card__price">$199</p>
+                  <h3 class="nft-card__title">NAME PRODUCT</h3>
+                </article>
+              </a>
+            </li>
+            <li class="nft-card">
+              <a href="#" class="nft-card__link">
+                <article class="nft-card__content">
+                  <img class="nft-card__image" src="#" alt="NFT Item 4">
+                  <span class="nft-card__badge nft-card__badge--new">NEW</span>
+                  <p class="nft-card__price">$199</p>
+                  <h3 class="nft-card__title">NAME PRODUCT</h3>
+                </article>
+              </a>
+            </li>
+            <li class="nft-card">
+              <a href="#" class="nft-card__link">
+                <article class="nft-card__content">
+                  <img class="nft-card__image" src="#" alt="NFT Item 5">
+                  <span class="nft-card__badge nft-card__badge--new">NEW</span>
+                  <p class="nft-card__price">$199</p>
+                  <h3 class="nft-card__title">NAME PRODUCT</h3>
+                </article>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <section class="blog">
+        <h2 class="blog__title">Our blog</h2>
+        <div class="blog__container">
+          <ul class="blog__list">
+            <li class="blog__item">
+              <img src="#" alt="Blog Post 1" class="blog__image" />
+              <span class="blog__tag">NFT</span>
+              <div class="blog__meta">
+                <p class="blog__author">Tony Nguyen</p>
+                <time datetime="2022-06-20" class="blog__date">20 Jun 2022</time>
+              </div>
+              <h3 class="blog__post-title">Sonic live</h3>
+              <p class="blog__excerpt">
+                One September 22, 2021 A volutpat nulla eros, nulla ultrices nec
+                nulla imperdiet...
+              </p>
+              <a href="#" target="_blank" class="blog__link">READMORE</a>
```

Comment:
BEM:

Pls use block class - `button` for all types of buttons in markup with different midifiers.

Reply:
Not fixed :(